specutils/io/default_loaders/tabular_fits.py

```diff
@@ -18,8 +17,12 @@
 
 
 def identify_tabular_fits(origin, *args, **kwargs):
-    # check if file can be opened with this reader
     # args[0] = filename
+    # check if filename conforms to naming convention for writer
+    if origin == 'write':
```

Comment:
@dhomeier would mind updating the file handling here as well?

Reply:
OK, looking at that line I first thought that you wanted to see the generalised file handling in the writing case as well. I frankly did not see an obvious way to auto-detect by content the format of a file that is yet to be written. In the case of an file handle that is opened for updating, we might assume that the format should match the previous content. Or in fact, in this particular case an hdulist that is passed for writing to the _primary_ HDU would indeed imply a `wcs` format, since tables can only be written to extensions. But currently neither writer even accepts an hdulist as destination, both only work on file paths...

That rant aside, for the `tabular_fits_loader`, yes, will do – I thought that had already been added in master at some point. But the `'write'` case also needs to be updated anyway to remove ambiguity in the naming scheme.